cli/src/pcluster/config/common.py

```diff
@@ -386,8 +393,14 @@ class BaseDeploymentSettings(Resource):
     configuration file deploys some Cfn infrastructure as well.
     """
 
-    def __init__(self, lambda_functions_vpc_config: LambdaFunctionsVpcConfig = None, **kwargs):
+    def __init__(
+        self, default_user_home: str = None, lambda_functions_vpc_config: LambdaFunctionsVpcConfig = None, **kwargs
+    ):
         super().__init__(**kwargs)
+        self.default_user_home = Resource.init_param(
```

Comment:
I think we can move this to the ClusterDeploymentSettings as this  is just the cluster setting and not for both Cluster and Image configs

Reply:
yes it should be there good catch!